app/views/movements/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="splash header">
+  <%= link_to root_path do %>
+    <%= image_tag("arrow-back.png", alt: "Arrow back icon") %>
+  <% end %>
+  <h2>NEW TRANSACTION</h2>
+  <%= image_tag("search.png", alt: "Search icon") %>
+</div>
+
+<%= form_with model: @movement, url: group_movements_path, local: true do |f| %>
+  <%= f.text_field :name, autofocus: true, placeholder: "Name" %>
+  <%= f.number_field :amount, autofocus: true, placeholder: "Amount" %>
+  <div class="select-category">
+    <%= f.label :group_ids, "Select Category: " %>
+    <%= f.collection_select(:group_ids, current_user.groups, :id, :name, { include_blank: true }, multiple: true) %>
+  </div>
+  <br>
```

Comment:
### Add a new transaction page

- [ ]  According to the project requirements, When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category and not the Home Page. Kindly correct this as well. ✍🏻 
- [ ] It's one of the project requirements that The user fills out a form to create a new transaction with these mandatory parameters:
* name (mandatory)
* amount (mandatory)
* categories (mandatory at least one). 
- [ ] Kindly add validation to the categories input select field to prevent any user to submit the form without selecting any category👍🏻